code/modules/clothing/under/accessories/storage.dm

```diff
@@ -46,6 +56,10 @@
 	..()
 	loc = master_item
 
+/obj/item/weapon/storage/internal/Destroy()
+	master_item = null
+	return ..()
+
 /obj/item/clothing/accessory/storage/webbing
 	name = "webbing"
 	desc = "Strudy mess of synthcotton belts and buckles, ready to share your burden."
```

Comment:
Can you make a list for this one with the guns that would reasonably fit on a webbing?

Currently they fit EVERYTHING.